app/navigation/nonsipp/MemberDetailsNavigator.scala

```diff
@@ -69,7 +69,7 @@ object MemberDetailsNavigator extends JourneyNavigator {
 
     case SchemeMembersListPage(srn, true) =>
       refineV[OneTo99](userAnswers.membersDetails(srn).length + 1).fold(
-        _ => controllers.routes.JourneyRecoveryController.onPageLoad(),
+        _ => employercontributions.routes.EmployerContributionsController.onPageLoad(srn, NormalMode),
```

Comment:
It should go to EmployerContributionsController instead of JourneyRecoveryController, if it go to JourneyRecoveryController tests are failing.

Reply:
The user shouldn't be able to hit this code as the yes no button isn't shown when 99 members have been added but they could still post to the endpoint. In that scenario I would just continue with the button as if they had pressed no rather than taking them to an error page. But tbh either is fine as it shouldn't be hit anyway